ft/loader/loader.cc

```diff
@@ -130,7 +128,8 @@ static int add_big_buffer(struct file_info *file) {
             newbuffer = true;
     }
     if (result == 0) {
-        int r = setvbuf(file->file, (char *) file->buffer, _IOFBF, file->buffer_size);
+        int r = setvbuf(
+            file->file->file, (char *)file->buffer, _IOFBF, file->buffer_size);
```

Comment:
> "file->file->file" is something. Any way to make it better?
I agree this looks weird. But what can we do with this? There are three different data structures for file description for three different logical layers. Changing the member names cause extra changes in the code, and this will be only for the certain line nice look. I think this is not reasonable. And besides our goal is not the current code refactoring, there are a lot of things to refactor, especially in bulk-loader. I would prefer to keep it as is, at least for now. The cast is fixed.